### PR TITLE
At/cb 190 create the credit limit modal for income

### PR DESCRIPTION
**Pull request by Angelbt12**
Create the credit limit modal for income
image

The taskk includes:

(1) desktop version

(2) mobile version

(3) loading version

(4) error version

### CODE REVIEW

**Review by novadeveloper14 — APPROVED** [review]

File: src/components/modals/IncomeModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+          </StyledContainerClose>
+        </Stack>
+
+        <StyledDivider />
+
+        {error ? (
+          <Stack direction="column" alignItems="center" padding="16px">
+            <Icon icon={<MdErrorOutline />} size="32px" appearance="danger" />
+            <Text size="large" weight="bold" appearance="danger">
+              Error cargando datos
```
Comment (carin91) [inline src/components/modals/IncomeModal/index.tsx:94]: Relocate this into a config file

File: src/components/modals/IncomeModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+            </Text>
+            <Text size="small" appearance="dark" textAlign="center">
+              No se pudieron cargar los datos. Por favor, intente nuevamente más
```
Comment (carin91) [inline src/components/modals/IncomeModal/index.tsx:97]: Relocate all of the data rendered into a config file

File: src/components/modals/IncomeModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+
+  useEffect(() => {
+    setTimeout(() => {
+      setError(false);
+      setLoading(false);
+    }, 2000);
+  }, []);
+
+  return createPortal(
+    <Blanket>
```
Comment (carin91) [inline src/components/modals/IncomeModal/index.tsx:69]: Refactor the component dividing it between index and interface

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/components/modals/IncomeModal/index.tsx (hunk 1 above)
Comment (Angelbt12) [reply src/components/modals/IncomeModal/index.tsx]: ok

File: src/components/modals/IncomeModal/index.tsx (hunk 2 above)
Comment (Angelbt12) [reply src/components/modals/IncomeModal/index.tsx]: ok

File: src/components/modals/IncomeModal/index.tsx (hunk 3 above)
Comment (Angelbt12) [reply src/components/modals/IncomeModal/index.tsx:58]: It's not very clear to me

**Review by carin91 — APPROVED** [review]

**Pull request merged**